src/Task/style.css

```diff
@@ -55,9 +66,23 @@
                 box-shadow: none;
             }
         }
+
+        &.is-extra {
+            & .ola_task-title-extra {
+                display: none;
+            }
+            & .ola_task-title-text {
+                @media (--to-width-1) {
+                    text-overflow: initial;
+                    overflow: auto;
+                    white-space: initial;
+                    padding-right: 0;
+                }
+            }
+        }
```

Comment:
The same here: we can remove `is-extra` class.

Reply:
done